app/constants/apis.js

```diff
@@ -0,0 +1,8 @@
+export const CATCHA_URL =
+  'https://fathomless-mountain-29102.herokuapp.com/v1.0/captcha';
```

Comment:
These are all testing URLs and will need to be replaced by each white-labeled app.